lib/tasks/export/analytics_bad_links_exporter.rake

```diff
@@ -0,0 +1,23 @@
+require 'local-links-manager/export/analytics_exporter'
+
+namespace :export do
+  namespace :google_analytics do
+    desc "Export bad links status to Google Analytics"
+    task "bad_links": :environment do
+      service_desc = "Export bad links status to Google Analytics"
+      begin
+        Rails.logger.info("Starting link exporter")
+        Services.icinga_check(service_desc, true, "Starting GA bad link exporter to")
```

Comment:
I feel like the description here is truncated.  "Starting GA bad link exporter to" what?